api/wrangler.toml

```diff
@@ -1,4 +1,7 @@
 name = "publisher-tools-api"
 main = "dist/index.js"

```

Comment:
I found this `dist/index.js` causes issues with `pnpm -C api dev`. If I replaced with `src/index.ts`, it worked fine.